test/spec/protractor_sync_sherman_test.ts

```diff
@@ -0,0 +1,44 @@
+//TODO:  This is a temporary test file to be removed before merged.
+//TODO: The contents of this file should move ino README.md when ready
```

Comment:
[] Move the accepted changes into the README.md and remove this file

Reply:
Ah, so you're using this to prove that it actually works before putting it into the README? Nice thinking.

Although I would have taken your word for it if you had opened the PR with this going into the README...

Reply:
heheh .  thanks Mindy!  right, that's basically it.